torch/csrc/utils/tensor_new.cpp

```diff
@@ -93,41 +101,75 @@ void maybe_initialize_cuda(const Device device) {
   }
 }
 
-Tensor dispatch_zeros(c10::DispatchKey dispatch_key, at::ScalarType scalar_type, const optional<Device>& device, IntArrayRef sizes) {
+Tensor dispatch_zeros(
+    c10::DispatchKey dispatch_key,
+    at::ScalarType scalar_type,
+    const optional<Device>& device,
+    IntArrayRef sizes) {
   maybe_initialize_cuda(dispatch_key);
   pybind11::gil_scoped_release no_gil;
   return torch::zeros(sizes, options(dispatch_key, scalar_type, device));
 }
 
-Tensor dispatch_ones(c10::DispatchKey dispatch_key, at::ScalarType scalar_type, const optional<Device>& device, IntArrayRef sizes) {
+Tensor dispatch_ones(
+    c10::DispatchKey dispatch_key,
+    at::ScalarType scalar_type,
+    const optional<Device>& device,
+    IntArrayRef sizes) {
   maybe_initialize_cuda(dispatch_key);
   pybind11::gil_scoped_release no_gil;
   return torch::ones(sizes, options(dispatch_key, scalar_type, device));
 }
 
-Tensor dispatch_full(c10::DispatchKey dispatch_key, at::ScalarType scalar_type, Scalar fill_value, const optional<Device>& device, IntArrayRef sizes) {
+Tensor dispatch_full(
+    c10::DispatchKey dispatch_key,
+    at::ScalarType scalar_type,
+    Scalar fill_value,
+    const optional<Device>& device,
+    IntArrayRef sizes) {
   maybe_initialize_cuda(dispatch_key);
   pybind11::gil_scoped_release no_gil;
-  return torch::full(sizes, fill_value, options(dispatch_key, scalar_type, device));
+  return torch::full(
+      sizes, fill_value, options(dispatch_key, scalar_type, device));
```

Comment:
@v0dro the changes in this file are pretty much all style-only changes that are unnecessary and unrelated to the purpose of this PR. The `clang-tidy` complaints are not interesting, but you wouldn't get them to begin with if you wouldn't make formatting changes like this.

I suggest to just undo them all. You want to keep the PR as small as possible to make it easy on reviewers. In the unlikely case you really _need_ style changes, it's best to make them in a separate PR so that that can be landed quickly.